### PR TITLE
Fix the invaild link of cmdlet in md file.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

Fix the invaild link of cmdlet in md file.
#12754

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafCustomRuleObject.md [hunk 1]
```diff
@@ -189,4 +189,4 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## RELATED LINKS
 
 [New-AzFrontDoorWafPolicy](./New-AzFrontDoorWafPolicy.md)
-[Set-AzFrontDoorWafPolicy](./Set-AzFrontDoorWafPolicy.md)
+[New-AzFrontDoorWafPolicy](./New-AzFrontDoorWafPolicy.md)
```
Comment (dingmeng-xue) [inline src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafCustomRuleObject.md:192]: should be Update-AzFrontDoorWafPolicy

File: src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafManagedRuleObject.md [hunk 2]
```diff
@@ -134,5 +134,5 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## RELATED LINKS
 
 [New-AzFrontDoorWafPolicy](./New-AzFrontDoorWafPolicy.md)
-[Set-AzFrontDoorWafPolicy](./Set-AzFrontDoorWafPolicy.md)
+[New-AzFrontDoorWafPolicy](./New-AzFrontDoorWafPolicy.md)
```
Comment (dingmeng-xue) [inline src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafManagedRuleObject.md:137]: Update-AzFrontDoorWafPolicy

File: src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafPolicy.md [hunk 3]
```diff
@@ -236,7 +236,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## RELATED LINKS
 
-[Set-AzFrontDoorWafPolicy](./Set-AzFrontDoorWafPolicy.md)
+[New-AzFrontDoorWafPolicy](./New-AzFrontDoorWafPolicy.md)
```
Comment (dingmeng-xue) [inline src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafPolicy.md:239]: Update-AzFrontDoorWafPolicy

File: src/Sql/Sql/help/New-AzSqlSyncGroup.md [hunk 4]
```diff
@@ -297,7 +297,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## RELATED LINKS
 
-[Set-AzSqlSyncGroup](./Set-AzSqlSyncGroup.md)
+[New-AzSqlSyncGroup](./New-AzSqlSyncGroup.md)
```
Comment (dingmeng-xue) [inline src/Sql/Sql/help/New-AzSqlSyncGroup.md:300]: Update-AzSqlSyncGroup

File: src/Sql/Sql/help/New-AzSqlSyncMember.md [hunk 5]
```diff
@@ -415,7 +415,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Get-AzSqlSyncMember](./Get-AzSqlSyncMember.md)
 
-[Set-AzSqlSyncMember](./Set-AzSqlSyncMember.md)
+[New-AzSqlSyncMember](./New-AzSqlSyncMember.md)
```
Comment (dingmeng-xue) [inline src/Sql/Sql/help/New-AzSqlSyncMember.md:418]: Update-AzSqlSyncMember

File: src/Resources/Resources/help/Remove-AzADUser.md [hunk 6]
```diff
@@ -246,5 +246,5 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Get-AzADUser](./Get-AzADUser.md)
 
-[Set-AzADUser](./Set-AzADUser.md)
+[New-AzADUser](./New-AzADUser.md)
```
Comment (dingmeng-xue) [inline src/Resources/Resources/help/Remove-AzADUser.md:249]: Update-AzADUser

File: src/Resources/Resources/help/Remove-AzADServicePrincipal.md [hunk 7]
```diff
@@ -288,7 +288,7 @@ Keywords: azure, azurerm, arm, resource, management, manager, resource, group, t
 
 [Get-AzADServicePrincipal](./Get-AzADServicePrincipal.md)
 
-[Set-AzADServicePrincipal](./Set-AzADServicePrincipal.md)
+[New-AzADServicePrincipal](./New-AzADServicePrincipal.md)
```
Comment (dingmeng-xue) [inline src/Resources/Resources/help/Remove-AzADServicePrincipal.md:291]: Update-AzADServicePrincipal

File: src/Resources/Resources/help/Remove-AzADApplication.md [hunk 8]
```diff
@@ -228,7 +228,7 @@ Keywords: azure, azurerm, arm, resource, management, manager, resource, group, t
 
 [Get-AzADApplication](./Get-AzADApplication.md)
 
-[Set-AzADApplication](./Set-AzADApplication.md)
+[New-AzADApplication](./New-AzADApplication.md)
```
Comment (dingmeng-xue) [inline src/Resources/Resources/help/Remove-AzADApplication.md:231]: Update-AzADApplication

File: src/Resources/Resources/help/New-AzADUser.md [hunk 9]
```diff
@@ -197,6 +197,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Get-AzADUser](./Get-AzADUser.md)
 
-[Set-AzADUser](./Set-AzADUser.md)
+[New-AzADUser](./New-AzADUser.md)
```
Comment (dingmeng-xue) [inline src/Resources/Resources/help/New-AzADUser.md:200]: Update-AzADUser

File: src/Resources/Resources/help/Get-AzADServicePrincipal.md [hunk 10]
```diff
@@ -273,7 +273,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [New-AzADServicePrincipal](./New-AzADServicePrincipal.md)
 
-[Set-AzADServicePrincipal](./Set-AzADServicePrincipal.md)
+[New-AzADServicePrincipal](./New-AzADServicePrincipal.md)
```
Comment (dingmeng-xue) [inline src/Resources/Resources/help/Get-AzADServicePrincipal.md:276]: needs to be corrected.

File: src/Resources/Resources/help/Get-AzADUser.md [hunk 11]
```diff
@@ -252,7 +252,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [New-AzADUser](./New-AzADUser.md)
 
-[Set-AzADUser](./Set-AzADUser.md)
+[New-AzADUser](./New-AzADUser.md)
```
Comment (dingmeng-xue) [inline src/Resources/Resources/help/Get-AzADUser.md:255]: needs to be corrected.

File: src/Resources/Resources/help/Get-AzADApplication.md [hunk 12]
```diff
@@ -250,7 +250,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Remove-AzADApplication](./Remove-AzADApplication.md)
 
-[Set-AzADApplication](./Set-AzADApplication.md)
+[New-AzADApplication](./New-AzADApplication.md)
```
Comment (dingmeng-xue) [inline src/Resources/Resources/help/Get-AzADApplication.md:253]: needs to be corrected.

File: src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafCustomRuleObject.md (hunk 1 above)
Comment (LucasYao93) [reply src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafCustomRuleObject.md]: Done

File: src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafManagedRuleObject.md (hunk 2 above)
Comment (LucasYao93) [reply src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafManagedRuleObject.md]: Done

File: src/Sql/Sql/help/New-AzSqlSyncGroup.md (hunk 4 above)
Comment (LucasYao93) [reply src/Sql/Sql/help/New-AzSqlSyncGroup.md]: Done

File: src/Resources/Resources/help/Remove-AzADApplication.md (hunk 8 above)
Comment (LucasYao93) [reply src/Resources/Resources/help/Remove-AzADApplication.md]: Done

File: src/Resources/Resources/help/Get-AzADServicePrincipal.md (hunk 10 above)
Comment (LucasYao93) [reply src/Resources/Resources/help/Get-AzADServicePrincipal.md]: Done

File: src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafPolicy.md (hunk 3 above)
Comment (LucasYao93) [reply src/FrontDoor/FrontDoor/help/New-AzFrontDoorWafPolicy.md]: Done

File: src/Sql/Sql/help/New-AzSqlSyncMember.md (hunk 5 above)
Comment (LucasYao93) [reply src/Sql/Sql/help/New-AzSqlSyncMember.md]: Done

File: src/Resources/Resources/help/New-AzADUser.md (hunk 9 above)
Comment (LucasYao93) [reply src/Resources/Resources/help/New-AzADUser.md]: Done

File: src/Resources/Resources/help/Get-AzADUser.md (hunk 11 above)
Comment (LucasYao93) [reply src/Resources/Resources/help/Get-AzADUser.md]: Done

**Review by dingmeng-xue — APPROVED** [review]

**Comment by dingmeng-xue** [comment]
Merge with admin privilege because Windows time out but this PR just changes readme file.

**Pull request merged**